src/code/audio/code_8019AF00.c

```diff
@@ -4350,9 +4523,51 @@ void Audio_ClearSariaBgmAtPos(Vec3f* pos) {
     }
 }
 
-u8 sBgmPlayers[2] = { SEQ_PLAYER_BGM_MAIN, SEQ_PLAYER_BGM_SUB };
-void Audio_SplitBgmChannels(s8 volumeSplit);
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/Audio_SplitBgmChannels.s")
+/**
+ * Turns on and off channels from both bgm players in a way that splits
+ * equally between the two bgm channels. Split based on note priority
+ */
+void Audio_SplitBgmChannels(s8 volumeSplit) {
+    u8 volume;
+    u8 notePriority;
+    u16 channelBits;
+    u8 sBgmPlayers[2] = { SEQ_PLAYER_BGM_MAIN, SEQ_PLAYER_BGM_SUB };
+    u8 i;
+    u8 channelIndex;
+
+    if ((Audio_GetActiveSequence(SEQ_PLAYER_FANFARE) == NA_BGM_DISABLED) &&
+        (Audio_GetActiveSequence(SEQ_PLAYER_BGM_SUB) != NA_BGM_ROMANI_RANCH)) {
+        for (i = 0; i < 2; i++) {
+            if (i == 0) {
```

Comment:
Enum worth it, or not?

Reply:
Not really, it's just looping through the elements of `sBgmPlayers`. Although I will add:

`for (i = 0; i < ARRAY_COUNT(sBgmPlayers); i++) {`